build.rs

```diff
@@ -0,0 +1,10 @@
+fn main() {
+    #[cfg(all(target_os = "linux", feature = "action-platform-info"))]
+    {
+        println!("cargo:rerun-if-changed=src/action/libc_version.c");
+
+        cc::Build::new()
+            .file("src/action/libc_version.c")
+            .compile("liblibc_version.a");
+    }
+}
```

Comment:
I am affraid build script is not an option — this does not play well with our internal build infrastructure (for `rrg-proto` I already have to do some awkward workarounds).

Reply:
Yes, we can scrap the build script now that the [libc::gnu_get_libc_version](https://docs.rs/libc/0.2.137/libc/fn.gnu_get_libc_version.html) function is available in the standard library.

Reply:
Yes, we can scrap the build script. The [libc::gnu_get_libc_version](https://docs.rs/libc/0.2.137/libc/fn.gnu_get_libc_version.html) function is now available in the standard library.